common/game/mode/fugitive/cop_car/CopCar.gd

```diff
@@ -40,6 +40,12 @@ func _ready():
 			if seat.is_driver_seat:
 				driver_seat = seat
 
+func get_history_heartbeat() -> FugitiveHistoryCarEntry:
+	var newEntry := load("res://common/game/mode/fugitive/history/FugitiveHistoryCarEntry.gd").new() as FugitiveHistoryCarEntry
+	newEntry.position = global_transform.origin
```

Comment:
Store this as a Vector2, grabbing the `global_transform.origin.x` and `global_transform.origin.z` fields

Reply:
Oh in there also: `FugitiveHistoryCarEntry.new()`